server/src/models/services/models.js

```diff
@@ -87,11 +87,21 @@ export default function (model_singleton) {
     standards: [ServiceStandardSchema],
     service_report: [ServiceReportSchema],
   });
+  const ServiceGeneralStats = mongoose.Schema({
+    num_of_services: { type: Number },
+  });
+  const ServiceTypeStats = mongoose.Schema({
+    ...bilingual_str("id"),
+    ...bilingual_str("label"),
+    value: { type: Number },
+  });
 
   model_singleton.define_model("ServiceReport", ServiceReportSchema);
   model_singleton.define_model("StandardReport", StandardReportSchema);
   model_singleton.define_model("ServiceStandard", ServiceStandardSchema);
   model_singleton.define_model("Service", ServiceSchema);
+  model_singleton.define_model("ServiceGeneralStats", ServiceGeneralStats);
```

Comment:
Are we actually populating the `ServiceGeneralStats` and `ServiceTypeStats`  models? I can't see them get used anywhere

Reply:
Nope they're not!